substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProviderBase.java

```diff
@@ -85,10 +96,14 @@ public abstract class NativeImageDebugInfoProviderBase {
     final HashMap<JavaKind, HostedType> javaKindToHostedType;
     private final Path cachePath = SubstrateOptions.getDebugInfoSourceCacheRoot();
 
-    public NativeImageDebugInfoProviderBase(NativeImageCodeCache codeCache, NativeImageHeap heap, NativeLibraries nativeLibs, HostedMetaAccess metaAccess) {
+    /*
+     * A factory used when deriving the calling convention for a specific method
```

Comment:
A leftover?

Reply:
Hmm, not sure how this got in there? I think it may have happened when I rebased. I'll delete it.

Reply:
Done.